python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -129,6 +144,10 @@ def optimize(self, model: nn.Module,
                  metric: Optional[Callable] = None,
                  direction: str = "max",
                  thread_num: Optional[int] = None,
+                 accelerator: Optional[Tuple[str]] = None,
+                 precision: Optional[Tuple[str]] = None,
+                 use_ipex: Optional[bool] = None,
```

Comment:
1. should we take `channels_last` here as one of the setting that could be specified like something stated in https://github.com/analytics-zoo/nano/issues/214 ?

Reply:
Three questions here:
1. Does all 4-dim input data are image data ? Or Does channels_last works for all 4-dim data?
2. We don't have fp32_channels_last now, to achieve this, one way is a new class which only do one thing : wrap channels_last; another approach might be achieve this in `PytorchIPEXJITModel`.
3. Do you mean make `channels_last` as a filter like `accelerator` and `precision` ?

Reply:
if channels_last can apply on openvino / onnxruntime